stdlib/bigarray.mli

```diff
@@ -298,6 +298,32 @@ module Genarray :
      is not in the range 0 to 16 inclusive, or if one of the dimensions
      is negative. *)
 
+  val init: ('a, 'b) kind -> 'c layout -> int array -> (int array -> 'a) ->
+            ('a, 'b, 'c) t
+  (** [Genarray.init kind layout dimensions f] returns a new Bigarray [b]
+      whose element kind is determined by the parameter [kind] (one of
+      [float32], [float64], [int8_signed], etc) and whose layout is
+      determined by the parameter [layout] (one of [c_layout] or
+      [fortran_layout]).  The [dimensions] parameter is an array of
+      integers that indicate the size of the big array in each dimension.
+      The length of [dimensions] determines the number of dimensions
+      of the Bigarray.
+
+      Each element [Genarray.get b i] is initialized to the result of [f i].
+      In other words, [Genarray.init kind layout dimensions f] tabulates
+      the results of [f] applied to the indexes of a new big array whose
+      layout is described by [kind], [layout] and [dimensions].  The index
+      array [i] may be shared and mutated between calls to f.
+
+      For instance, [Genarray.init int c_layout [|2; 1; 3|]
+      (Array.fold_left (+) 0)] returns a fresh big array of integers, in C
+      layout, having three dimensions (2, 1, 3, respectively), with the
+      element values 0, 1, 2, 1, 2, 3.
+
+      [Genarray.init] raises [Invalid_argument] if the number of dimensions
+      is not in the range 0 to 16 inclusive, or if one of the dimensions
+      is negative. *)
+
```

Comment:
"since" annotation missing (here and below).

Reply:
Thanks (29e78e7).